tfhe/src/shortint/server_key/mod.rs

```diff
@@ -398,6 +398,24 @@ impl ServerKey {
         }
     }
 
+    pub(crate) fn generate_lookup_table_no_encode<F>(&self, f: F) -> LookupTableOwned
+    where
+        F: Fn(u64) -> u64,
+    {
+        let mut acc = GlweCiphertext::new(
+            0,
+            self.bootstrapping_key.glwe_size(),
+            self.bootstrapping_key.polynomial_size(),
+            self.ciphertext_modulus,
+        );
+        fill_accumulator_no_encoding(&mut acc, self, f);
+
+        LookupTableOwned {
+            acc,
+            degree: Degree::new(self.message_modulus.0 * self.carry_modulus.0 * 2),
```

Comment:
I don't think that degree is correct ?

Reply:
We should not rely on the degree in this case IMO
So better give the biggest value

Reply:
it's updated anyways for the output of the PRF as far as I can see, just put the comment you just made there in the code then :)

Reply:
Done